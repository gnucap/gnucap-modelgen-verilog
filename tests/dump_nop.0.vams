
`include "sub/disciplines.few"

module test(d, g, s, b);
	electrical d, g;
	real T;
	real U;
	analog begin
		T = V(d, g) * 0.;
		T = 0. * V(d, g);
		T = 0. * V(d, g) * exp(T);
		T = 0. * exp(17.);
		T = exp(17.) * 0.;
		T = exp(17.) * (0. + 0.);
		T = exp(17.) * (0. * 0.);
		T = (0. * 0.) * exp(42.);
		T = exp(T) * ln(1);
		T = ln(1) * exp(T);
		T = 0. * exp(V(d, g)+4);
		T = exp(V(d, g)+4) * 0.;
	end
endmodule





