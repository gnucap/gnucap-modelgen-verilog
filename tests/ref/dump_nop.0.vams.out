nature current
  access = "I";
  units = "A";
  abstol = "0.001234";
endnature

nature voltage
  access = "V";
  units = "V";
  abstol = "0.0004321";
endnature


discipline electrical
  potential "voltage";
  flow "current";
enddiscipline


module test(d, g, s, b);
  electrical d, g;
  real   T;
  real   U;

  analog begin
    T = 0.;
    T = 0.;
    T = 0.;
    T =  0.;
    T =  0.;
    T =  0.;
    T =  0.;
    T =  0.;
    T =  0.;
    T =  0.;
    T = 0.;
    T = 0.;
  end

endmodule



